src/literals.rs

```diff
@@ -15,8 +15,13 @@ use memchr::{memchr, memchr2, memchr3};
 use syntax;
 
 use freqs::BYTE_FREQUENCIES;
+
+#[cfg(feature = "simd-accel")]
```

Comment:
I think you don't actually need this `cfg` switch, right? The `cfg` switch is done in `lib.rs` so that `simd_fallback` should never actually be explicitly used.